package/MDAnalysis/lib/formats/include/readdcd.h

```diff
@@ -136,7 +136,7 @@ static int write_dcdheader(fio_fd fd, const char *remarks, int natoms,
  * Output: 0 on success, negative error code on failure.
  * Side effects: coordinates are written to the dcd file.
  */
-static int write_dcdstep(fio_fd fd, int curstep, int curframe, 
```

Comment:
If this is a change with upstream that it OK. i didn't know that sorry.

I have no idea how we got that file. I didn't actually know that it is from VMD. I then also stuck with it. If there is a newer version which we can include I'm for it.